ace-modes.d.ts

```diff
@@ -1,1149 +1,1683 @@
-  export const SoyTemplateHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const SoyTemplateHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/space" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/space_highlight_rules" {
-  export const SpaceHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const SpaceHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/sparql" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/sparql_highlight_rules" {
-  export const SPARQLHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const SPARQLHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/sql" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
-}
-declare module "ace-code/src/mode/sql_highlight_rules" {
-  export const SqlHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/sqlserver" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/sqlserver_highlight_rules" {
-  export const SqlHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const SqlHighlightRules: new () => import(".").Ace.HighlightRules;
+}
+
+declare module "ace-code/src/mode/sql_highlight_rules" {
+    export const SqlHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/stylus" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/stylus_highlight_rules" {
-  export const StylusHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const StylusHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/svg" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/svg_highlight_rules" {
-  export const SvgHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const SvgHighlightRules: new () => import(".").Ace.HighlightRules;
 }
+
 declare module "ace-code/src/mode/swift" {
-  export const Mode: new () => import(".").Ace.SyntaxMode;
+    export const Mode: new () => import(".").Ace.SyntaxMode;
 }
+
 declare module "ace-code/src/mode/swift_highlight_rules" {
-  export const SwiftHighlightRules: new () => import(".").Ace.HighlightRules;
+    export const HighlightRules: new () => import(".").Ace.HighlightRules;
```

Comment:
Now that I added `SwiftHighlightRules` in #5516, which one should we export here?

Reply:
Currently, the script adds both types of exports. It collects all exports and generates declarations based on them. I don’t see this as a problem, especially considering that it accurately reflects the current codebase. However, if desired, I could add an exclusion list for exports we prefer not to include.

Reply:
Oh I see, I've just merged the change after this PR is created and couldn't see it reflected. Will it be included in the next time we run this script?

Reply:
Yes, I merged new changes and run script again -> now there are two exports